cocos/3d/CCTerrain.h

```diff
@@ -304,6 +304,9 @@ class CC_DLL Terrain : public Node
     friend QuadTree;
     friend Chunk;
 public:
+    void setLightMap(const std::string& fileName);
+
+    void setLightDir(const Vec3& lightDir);
```

Comment:
It is in the terrain's local space. Terrain usually placed at (0, 0, 0) and without rotation, so most of the time we can say it is in world space.